jsonschema/src/fields.ts

```diff
@@ -201,8 +202,8 @@ export type OpeningHours = string;
 /**
  * A field for the schema version.
  */
-export type SchemaVersion = `${number}.${number}`;
+export type SchemaVersion = `0.2`;
```

Comment:
Did you mean to sub this type alias? Not sure if you wanted to keep the template type or use it as a constant (as below for `SchemaID`).

```suggestion
export const SchemaVersion = '0.2';
```

Reply:
Changed to a constant consistent with other types. If we do things right, next iteration of the schema should be compatible with 0.2 and 0.3 hence the need for a type.

Reply:
Rebuilt the schema and as expected this change has no impact on the resulting schema.

Reply:
Sorry, didn't mean to be nitpicky! I just wanted to make sure the change was intentional. It looked like `0.2` might have been an accidental search/replace/template substitution/etc. of the generic type expression.